tests/test_backend.py

```diff
@@ -22,32 +22,48 @@ def backend(request):
     [
         (
             [],
-            'empty',
+            'empty.zip',
             None,
             '1.0.0',
             None,
         ),
         (
             'file.ext',
-            'not-empty',
+            'not-empty.zip',
             None,
             '1.0.0',
             None,
         ),
         (
             ['file.ext', 'dir/to/file.ext'],
-            'not-empty',
+            'not-empty.zip',
             'group',
             '1.0.0',
             None,
         ),
         (
             ['file.ext', 'dir/to/file.ext'],
-            'not-empty',
+            'not-empty.zip',
             'group',
             '2.0.0',
             'tmp',
         ),
+        (
+            ['file.ext', 'dir/to/file.ext'],
+            'not-empty.tar.gz',
+            'group',
+            '2.0.0',
+            'tmp',
+        ),
+        # extension not supported
+        pytest.param(
+            ['file.ext', 'dir/to/file.ext'],
+            'not-supported.bad',
+            'group',
+            '2.0.0',
+            'tmp',
+            marks=pytest.mark.xfail(raises=RuntimeError),
```

Comment:
I think this tests only the wrong extension when trying to put an archive to the backend, not getting it. For the later you need to add something like:

```python
audeer.touch('archive.yip')
backend.put_file('archive.yip', 'archive.yip', '1.0.0')
backend.get_archive('archive.yip', 'download', version='1.0.0')
```